### DISCUSSION TITLE
It suddenly stops working for student plan

### DISCUSSION THREAD

**Discussion by dibasdauliya** [community/community#42234] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

It is saying "subscription required", and when I click there it says "You are eligible to use GitHub Copilot for free.". But it is not working at all.

I tried reloading, signing out and signing in, reverted to old version. But it is still not working.

![image](https://user-images.githubusercontent.com/51583004/208757798-2b5efe85-d8ec-49a6-a63e-95d9b461cdd9.png)

![image](https://user-images.githubusercontent.com/51583004/208757877-ec052075-2cc6-4998-8036-a074f82e990b.png)

**Comment by lacker** [comment]
I have the same problem - in the extension it shows "subscription required", I click on it, and it just takes me to the billing page saying I am eligible to use it for free. Tried updating VS Code and the extension, tried logging out and logging back in to Github via VSCode, still doesn't work. This is on Ubuntu fwiw.

**Reply by lacker** [reply]
I tried this on two separate machines - on a Windows machine, Copilot was working fine last week, and now when I return to the same development environment, Copilot is no longer working. Maybe it had updated some VSCode or Copilot extension, I'm not sure. On the Ubuntu machine, it's a completely new setup, which doesn't work. Both have the same problem - it says "subscription required", and clicking that sends me to a page which tells me I should already have access.

**Comment by iamcjwilliams** [comment]
@dibasdauliya Sorry for the trouble here, please visit [here](https://education.github.com/discount_requests/pack_application) in order to signup for free student access. If you are still experiencing issues please submit a support ticket here: [Support page](https://support.github.com/contact).
